### ISSUE TITLE
Is it possible to highlight lines one by one?

### ISSUE THREAD

**Issue by KnockOutEZ** [taoky/pdf-annotate.js#22]
I want to generate the coordinates of each line so I can use them in the Highlight annotation:
![image](https://github.com/taoky/pdf-annotate.js/assets/70368615/f3f8746a-d469-4f2c-a75c-1946829249c9)
I want to iterate one after another line and highlight them. How can I achieve this?

I also wanna add another question which is, can I trigger a function on line click? For example I click on line 100 in whole pdf, and it triggers a function returning 100 (The index of the line in whole pdf).

**Comment by taoky** [comment]
I don't think this works as there is no such concept of "line number" or "line" in PDF, as it is originally a complex format designed for printing instead of editing.
